From 0cc3dd71213687ccb5ec0a1866864558bf03260b Mon Sep 17 00:00:00 2001
From: Stanislav Ochotnicky <sochotnicky@redhat.com>
Date: Fri, 7 May 2010 14:45:44 +0200
Subject: [PATCH] Fix qt4 pinentry window created in the background

This is probably just a workaround. Proper fix is being investigated.
---
 qt4/pinentrydialog.cpp |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/qt4/pinentrydialog.cpp b/qt4/pinentrydialog.cpp
index 8644136..6064ab9 100644
--- a/qt4/pinentrydialog.cpp
+++ b/qt4/pinentrydialog.cpp
@@ -63,7 +63,6 @@ void raiseWindow( QWidget* w )
     SetForegroundWindowEx( w->winId() );
 #endif
     w->raise();
-    w->activateWindow();
 }
 
 QPixmap icon( QStyle::StandardPixmap which )
-- 
1.6.6.1

